lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -2337,13 +2337,13 @@ namespace Js
             return si;
     }
 
-#ifdef ENABLE_GLOBALIZATION
     void ScriptContext::UpdateTimeZoneInfo()
     {
         GetTimeZoneInformation(&timeZoneInfo);
+#if !defined(__cplusplus_winrt) && defined(_WIN32)
```

Comment:
Saw the `important` message from https://msdn.microsoft.com/en-us/library/90s5c885.aspx

AFAIK using `tzset` on *nix may depend on the choice of localization.